zzzz_modular_occulus/code/modules/mob/living/simple_animal/hostile/siren/replicant.dm

```diff
@@ -272,7 +237,7 @@
 	icon_aggro = "Replicanttendril"
 	icon_dead = "Replicanttendrildead"
 	mouse_opacity = 2
-	move_to_delay = 0
```

Comment:
the move delay of zero was, afaik, intentional. To simulate the replicant lashing people across the room with tendrils that can be shot.

Reply:
A move delay of zero = Ian looking at food speeds, with *several* of these being spawned in at once.
Probably somewhere around 4-5 at once, I think?
They do ~15 damage per hit, so all that summed up seems pretty unbalanced

Reply:
It has already been tested once and it was not imbalanced. Kindly put it back.

Everything else is fine.

Reply:
Correct. Tendrils have a bit of delay after spawning to move, and a delay after moving before they attack.

The reason being what Jamini listed. The tendrils only spawn each time a replicant attacks.